requirements.txt

```diff
@@ -1,5 +1,9 @@
+opencv-python==4.1.1.26
+pafy==0.5.5
 pandas>=0.25.2
+PyYAML==5.1.2
```

Comment:
Same here, lower-bound version requirement would be better :)

Reply:
Done!